sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/unit/tests/inputs/DBInputWritable.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.beam.sdk.io.hadoop.inputformat.unit.tests.inputs;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.apache.beam.sdk.io.hadoop.inputformat.integration.tests.HIFIOWithPostgresIT;
+import org.apache.hadoop.io.Writable;
+import org.apache.hadoop.mapreduce.lib.db.DBWritable;
+
+/**
+ * To access the data from RDBMS using {@link DBInputFormat}, you have to create a class to define
+ * the data which you are going to read. {@link DBInputWritable} is a class to read data from
+ * Postgres DB in test {@link HIFIOWithPostgresIT}. {@link DBInputWritable} holds id and name of the
```

Comment:
I don't see HIFIOWithPostgresIT in this PR - if it's not included in this PR, maybe you should remove this file from this PR and include it where you have HIFIOWithPostgresIT?